Cython/Utility/ObjectHandling.c

```diff
@@ -2498,6 +2502,9 @@ static PyObject *__Pyx_PyFunction_FastCallDict(PyObject *func, PyObject **args,
 #endif
   #define __Pxy_PyFrame_Initialize_Offsets()
   #define __Pyx_PyFrame_GetLocalsplus(frame)  ((frame)->f_localsplus)
+#if PY_VERSION_HEX >= 0x030D0000
+  #include "internal/pycore_unicodeobject.h"
+#endif
```

Comment:
We generally try to avoid including internal CPython headers.

Reply:
This is not needed any more, I missed removing this when I version-gated farther down in the code.